File: python/7 kyu/313.py
# https://www.codewars.com/kata/5f77d62851f6bc0033616bd8/train/python

# Passed

def valid_spacing(s):
    split_terms = s.split(' ') if s else []
    result = all(term != '' for term in split_terms)
    return result

output = valid_spacing('Hello world')
print(output)

File: nodejs/7 kyu/313.js
// https://www.codewars.com/kata/560d6ebe7a8c737c52000084/train/javascript

// Passed

function notVisibleCubes(n) {
  let result = n <= 2 ? 0 : (n - 2) ** 3;
  return result;
}

const output = notVisibleCubes(5);
console.log(output);